src/lib/core/api/analysis-api.ts

```diff
@@ -61,15 +63,30 @@ export class AnalysisClient extends FetchClientWithCredentials {
     }
   }
 
-  private findUserPath(userId: number) {
+  protected async fetchWithDetails<T>(
+    callback: (user: User, projectId: string) => ApiRequest<T>
+  ): Promise<T> {
+    const projectId = (await this.wdkService.getConfig()).projectId;
+    return this.fetchWithUser((user) => callback(user, projectId));
+  }
+
+  private makeUserPath(userId: number) {
     return `/users/${userId}`;
   }
 
+  private makePreferencesPath(userId: number, projectId: string) {
+    return `${this.makeUserPath(userId)}/preferences/${projectId}`;
+  }
+
+  private makeAnalysesPath(userId: number, projectId: string) {
+    return `${this.makeUserPath(userId)}/${projectId}/analyses`;
```

Comment:
This should now be reversed to match the re-edited user service.  /users/{user-id}/analyses/{project-id}